microservices/embeddingservice/src/embed_text.py

```diff
@@ -69,8 +69,18 @@ def __init__(self, config):
                             local_path=local_path,
                         )
                 load_path = local_path if os.path.exists(local_path) else model_path
+
+                # needed to get the correct weights for the jina model
+                if local_path:
+                    trust_remote_code = True
+                else:
+                    trust_remote_code = False
+
```

Comment:
Ok, tatsächlich verstehe ich nicht, unter welchen Umständen wir hier in den "else" Path laufen, und was es uns überhaupt bringen sollte, jemals  "trust_remote_code=false" an den Konstruktor zu übergeben. Wenn ich Dich recht verstehe, erzeugtt das MOdell "kaputte" Embeddings, wenn wir  "trust_remote_code=false" übergeben - dann nützt uns das ganze ja auch nichts, oder?